src/librustc/hir/map/mod.rs

```diff
@@ -330,12 +330,12 @@ impl<'hir> Map<'hir> {
             },
             Node::TraitItem(item) => match item.kind {
                 TraitItemKind::Const(..) => DefKind::AssocConst,
-                TraitItemKind::Method(..) => DefKind::Method,
+                TraitItemKind::Method(..) => DefKind::AssocFn,
```

Comment:
We should also rename the `Method` variants to `Fn` as well.

Reply:
That's also included in #60163.